flask_resty/api.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, app=None, prefix=''):
             self._app = app
             self.init_app(app)
 
+        # store the view urls for reference. Doesn't support multiple routes
+        # mapped to same view
+        self.views = {}
```

Comment:
Have to check if I have the reference to the app in the location I have to use it - but I think so